src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DeclarationOrderCheck.java

```diff
@@ -84,41 +84,112 @@
  * </li>
  * </ul>
  * <p>
- * To configure the check:
+ * To configure the check with default config:
  * </p>
  * <pre>
  * &lt;module name=&quot;DeclarationOrder&quot;/&gt;
  * </pre>
- * <p>
- * With default options:
- * </p>
+ * <p>Example:</p>
  * <pre>
- * class K {
- *   int a;
- *   void m(){}
- *   K(){}  &lt;-- &quot;Constructor definition in wrong order&quot;
- *   int b; &lt;-- &quot;Instance variable definition in wrong order&quot;
+ * public class Test {
+ *
+ *   public int a;
+ *   protected int b;
+ *   public int c;            // violation, variable access definition in wrong order
+ *
+ *   Test() {
```

Comment:
it may be useful to add different modifiers to constructors to show difference as well

Reply:
I experimented with modifiers on constructors in a different source file, and check does not raise violation like such "constructor access definition in wrong order" for constructors. Should I add this in a para below the example?

Reply:
if there is no violation raised, it is not needed